doc/how_to_contribute/how_to_contribute_to_site.rst

```diff
@@ -25,59 +26,61 @@ This should make it easy for you to develop changes to this site and make sure y
 
```

Comment:
Do the sentences above this need to be here? Seems like this would fit better in the intro.

Reply:
What do you think about removing them entirely?  It looks like this information is already covered in the "Learning Objectives" section.

Reply:
Sounds good to me!

Reply:
Done.